plugins/source/azuredevops/go.mod

```diff
@@ -4,6 +4,7 @@ go 1.19
 
 require (
 	github.com/cloudquery/plugin-sdk v1.12.0
+	github.com/microsoft/azure-devops-go-api/azuredevops v1.0.0-b5
```

Comment:
https://pkg.go.dev/github.com/microsoft/azure-devops-go-api/azuredevops/v6

Reply:
Good catch done in https://github.com/cloudquery/cloudquery/pull/5611/commits/0e33317e24f44fc4e466ef167b661b9147cd47a1.

I guess they don't bother updating the readme https://github.com/microsoft/azure-devops-go-api/tree/00dac5c867394a3c5ca4e12b6965d7625a1588c6#azure-devops-go-api